src/index.test.js

```diff
@@ -1,7 +1,12 @@
+/* eslint-disable func-names, no-console */
+
 import expect from 'expect.js'
+import {getSupport, currentBrowser} from 'caniuse-support'
 
 import {prefix, supportedProperty, supportedValue} from './index'
 
+console.log(`Detected browser: ${currentBrowser.id} ${currentBrowser.version}`)
```

Comment:
maybe disbable eslint just for this line, because its very likely that we forget console.log statements after debugging and now lint won't warn.